### PR TITLE
MAT-7793: Apply Function

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-7793](https://jira.cms.gov/browse/MAT-7793)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/editor/__mocks__/cqlFunctionApplierTest.cql [hunk 1]
```diff
@@ -0,0 +1,33 @@
+library MAT6197UCUMAnnotations version '0.0.000'
```
Comment (sb-cecilialiu) [inline src/components/editMeasure/editor/__mocks__/cqlFunctionApplierTest.cql]: is this file used anywhere?

File: src/components/editMeasure/editor/cqlFunctionAppliernoParams.cql [hunk 2]
```diff
@@ -0,0 +1,16 @@
+library T1 version '16.0.000'
```
Comment (sb-cecilialiu) [inline src/components/editMeasure/editor/cqlFunctionAppliernoParams.cql]: should this be under __mocks__?

File: src/components/editMeasure/editor/__mocks__/cqlFunctionApplierTest.cql (hunk 1 above)
Comment (mcmcphillips) [reply src/components/editMeasure/editor/__mocks__/cqlFunctionApplierTest.cql]: No, deleted

File: src/components/editMeasure/editor/cqlFunctionAppliernoParams.cql (hunk 2 above)
Comment (mcmcphillips) [reply src/components/editMeasure/editor/cqlFunctionAppliernoParams.cql]: It should if there wasn't already another copy of the same file. 
Deleted duplicate

**Review by sb-cecilialiu — APPROVED** [review]
Looks good to me.

**Pull request merged**